web/src/app/layout.tsx

```diff
@@ -14,9 +14,9 @@ export default function RootLayout({ children }: { children: React.ReactNode })
     <html lang="en">
       <body>
         <AuthProvider>
-          <div className="layout">
+          <div className="content__wrapper">
```

Comment:
I don't think that's correct BEM, because your `content` class here is inside `content__wrapper`, it should be the other way around. Your `content` is the block and your `content__wrapper` is the element within the block. Unless I'm reading it wrong?

Reply:
I always read wrappers as the outside; when you have a candy, the candy is the content, the wrapper is the content wrapper

Reply:
Yeah, but that's not how BEM works 😅

Reply:
hmmm, will have a think and I guess overhaul because content__content doesn't seem like a good choice